src/js/disability-benefits/containers/StatusPage.jsx

```diff
@@ -7,15 +7,15 @@ import AskVAToDecide from '../components/AskVAToDecide';
 import ClaimsTimeline from '../components/ClaimsTimeline';
 import ClaimDetailLayout from '../components/ClaimDetailLayout';
 import { setUpPage, isTab, scrollToTop, setFocus } from '../utils/page';
-import { itemsNeedingAttentionFromVet, getCompletedDate } from '../utils/helpers';
+import { itemsNeedingAttentionFromVet, getClaimType, getCompletedDate } from '../utils/helpers';
 
 import { clearNotification } from '../actions';
 
 const FIRST_GATHERING_EVIDENCE_PHASE = 3;
 
 class StatusPage extends React.Component {
   componentDidMount() {
-    document.title = 'Status - Your Disability Compensation Claim';
+    document.title = `Status - Your ${getClaimType(this.props.claim)} Claim`;
```

Comment:
This is throwing errors in the console, the claim may not be loaded when the component is mounted (this is also true on the details and files tabs, but those are typically accessed after this tab). I think a generic title here might be best, and possibly update it when the props change. @jbalboni Do you have any suggestions on handling this case?